.github/workflows/commitlint.yml

```
@@ -19,10 +19,14 @@ jobs:
       - name: Install dependencies
         run: npm install commitlint @commitlint/config-conventional
       - name: Run commitlint against commits in PR
+        env:
+          WORKSPACE: ${{ github.workspace }}
+          BASE_SHA: ${{ github.event.pull_request.base.sha }}
+          HEAD_SHA: ${{ github.event.pull_request.head.sha }}
         run: |
           npx commitlint \
-            -g ${{ github.workspace }}/.commitlint.config.mjs \
-            --from ${{ github.event.pull_request.base.sha }} \
-            --to ${{ github.event.pull_request.head.sha }} \
+            -g "$WORKSPACE"/.commitlint.config.mjs \
```

nit: we should probably use `{}` for this and quote the whole string.
```suggestion
            -g "${WORKSPACE}/.commitlint.config.mjs" \
```
It's less important for the others since its a single variable but consistency for all of them would be nice.